packages/lib-classifier/src/store/SubjectStore.js

```diff
@@ -18,7 +18,7 @@ const MINIMUM_QUEUE_SIZE = 3
 
 const SingleSubject = types.union(SingleImageSubject, SingleVideoSubject, Subject)
 function subjectDispatcher (snapshot) {
-  if (snapshot.subjects) {
+  if (snapshot.metadata['#subject_group_id']) {
```

Comment:
It might be possible to write a test for this, by doing something like checking that `getType(subjects.active)` returns a `SubjectGroup` and not a `Subject`.

Reply:
I've added existence checks here, just to be safe. Following up with your suggestion for a getType() test.

Reply:
Testes added 👍

Reply:
TESTS ADDED, GODDAMMIT